Source/CppConsoleTable.hpp

```diff
@@ -0,0 +1,584 @@
+/*
```

Comment:
Why are you including this file in /Source? This file should be referenced as a header include path from the git submodule in /Vendor.

Reply:
Yep, with the backslashes replaced in the .xcodeproj file this is indeed unnecessary!